tools/src/main/java/org/apache/kafka/trogdor/workload/ProduceBenchWorker.java

```diff
@@ -273,6 +338,7 @@ StatusData update() {
         private final int p50LatencyMs;
         private final int p95LatencyMs;
         private final int p99LatencyMs;
+        private final int transactionsCommitted ;
```

Comment:
This is a checkstyle issue.  Are you compiling with `check` to catch these before Jenkins does?